### PR TITLE
[ALLUXIO-2351] Update language in Swift docs

### PR DESCRIPTION
**Pull request by quq99**
https://alluxio.atlassian.net/browse/ALLUXIO-2351
Update language in Swift docs

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Jenkins, add to whitelist

File: docs/en/Configuring-Alluxio-with-Swift.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ group: Under Store
 priority: 1
 ---
 
-This guide describes how to configure Alluxio with
-[Swift](http://docs.openstack.org/developer/swift/) as the under storage system.
+This guide describes how to configure Alluxio with an under storage system
+supporting the [Swift](http://docs.openstack.org/developer/swift/).
```
Comment (aaudiber) [inline docs/en/Configuring-Alluxio-with-Swift.md]: I think this should be `Swift API` instead of just `Swift`

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
@quq99 Thanks for the PR! Could you resolve the conflicts?

File: docs/en/Configuring-Alluxio-with-Swift.md (hunk 1 above)
Comment (quq99) [inline docs/en/Configuring-Alluxio-with-Swift.md]: Thanks @aaudiber .
I have changed it to Swift API.

**Pull request merged**